src/labels/CustomLabels.labels

```diff
@@ -13910,4 +13910,84 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
         <shortDescription>Video header title in Get Started</shortDescription>
         <value>Get Started on NPSP</value>
     </labels>
+    <labels>
+        <fullName>gseuResourcesItemQuestionsAnsweredLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Questions Answered label in GS Resources section for End User</shortDescription>
+        <value>Get Your Questions Answered</value>
+    </labels>
+    <labels>
+        <fullName>gseuResourcesItemQuestionsAnsweredLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource Questions Answered link label in GS Resources section for End User</shortDescription>
+        <value>Power of Us Hub</value>
+    </labels>
+    <labels>
+        <fullName>gseuResourcesItemAboutNPSPLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Resource About NPSP label in GS Resources section for End User</shortDescription>
```

Comment:
Please change to: About NPSP label in Get Started Resources section for End Users.